executor/src/lib.rs

```diff
@@ -463,7 +463,7 @@ mod tests {
 		(block1, block2)
 	}
 
-	fn block_with_size(time: u64, nonce: u32, size: usize) -> (Vec<u8>, Hash) {
+	fn block_with_size(time: u64, nonce: u64, size: usize) -> (Vec<u8>, Hash) {
```

Comment:
how about 
```suggestion
        use cennznet_primitives::{Index, Timestamp};
	fn block_with_size(time: Timestamp, nonce: Index, size: usize) -> (Vec<u8>, Hash) {
```?

Reply:
Ah yep sounds good! I'll make the change